### PR TITLE
Legal updates from CELA

### PR DESCRIPTION
**Pull request by Taojunshen**
CELA has provided replacement licence files which include separate licenses for CC and code snippets. The Content Engineering Team has been tasked with replacing the license files on all existing repos. The changes to the README file must be at the top of the file to be recognized by license scanners.
 
Please see this communication for more details: [Legal Update Notice](https://microsoft.sharepoint.com/teams/STBCSI/e/CE/_layouts/15/guestaccess.aspx?guestaccesstoken=m4R%2fQ5Es3YlfV%2fUTJ09pb8x50MXjJNNO20%2b9YHpi5%2bo%3d&docid=2_0a46b1b0a931049e5afab07102ab5c0df&rev=11)
 
Please accept the PR as soon as possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## Microsoft Open Source Code of Conduct
```
Comment (mairaw) [inline README.md]: this project is under  the .NET Foundation Code of Conduct which is called out below. see https://opensource.microsoft.com/codeofconduct/faq/#alreadyhave

**Review by mairaw — CHANGES_REQUESTED** [review]
left a comment about the code of conduct piece. also, shouldn't the files be markdown so links and headers would render correctly?

**Comment by v-anpasi** [comment]
@mairaw, Does .NET fall under APEX management? If not, this PR can be closed and ignored. Otherwise, I will have to look into it further. We were given a list of repos to update and are running through our process. Thanks for working with us. ^_^

**Comment by mairaw** [comment]
@v-anpasi yes, it's under APEX management but since our project is in a different organization it follows our own code of conduct.

File: README.md (hunk 1 above)
Comment (v-anpasi) [inline README.md]: Fixed

**Review by mairaw — APPROVED** [review]

**Pull request merged**